### PR TITLE
Task7

### PR DESCRIPTION
**Pull request by iharyachnik**

### CODE REVIEW

File: controllers/cities/cities.service.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { cities } from '../../models';
+
+class CitiesService {
+  getCities = () => cities.find();
+
+  getRandomCityManually = () => {
```
Comment (DraXu3) [inline controllers/cities/cities.service.js:6]: Why to define this function in such way if it was already defined below?

File: utils/validators.js [hunk 2]
```diff
@@ -0,0 +1 @@
+export const firstLetterCaseValidator = value => value[0] === value[0].toUpperCase();
```
Comment (DraXu3) [inline utils/validators.js]: Won't work if `value` will be an empty string

**Review by DraXu3 — COMMENTED** [review]
Please remove redundant function and update one in utils

File: controllers/cities/cities.service.js (hunk 1 above)
Comment (iharyachnik) [reply controllers/cities/cities.service.js:6]: Because as I understood task required 2 different implementations using built-in tools and manually

**Pull request merged**